data/data/aws/main.tf

```diff
@@ -66,12 +67,14 @@ module "masters" {
   ec2_ami                  = var.aws_region == var.aws_ami_region ? var.aws_ami : aws_ami_copy.imported[0].id
   user_data_ign            = var.ignition_master
   publish_strategy         = var.aws_publish_strategy
+  iam_role_name            = var.aws_master_iam_role_name
 }
 
 module "iam" {
   source = "./iam"
 
-  cluster_id = var.cluster_id
+  cluster_id      = var.cluster_id
+  worker_iam_role_name   = var.aws_worker_iam_role_name
```

Comment:
These need to be aligned on the `=`.
Run `./hack/tf-fmt.sh` to see the formatting violations.
You can also run the following to make the format changes.
```
podman run --rm \
 --env IS_CONTAINER=TRUE \
 --volume "${PWD}:${PWD}:z" \
 --workdir "${PWD}" \
 quay.io/coreos/terraform-alpine:v0.12.0-rc1 \
 terraform fmt -write=true -recursive data/data
```